### ISSUE TITLE
Custom control with a Popup in the ControlTemplate does not display in Wasm

### ISSUE THREAD

**Issue by jsinsel** [unoplatform/uno#4660]
## Current behavior

When creating a custom control that has a Popup in the ControlTemplate, it does not display in Wasm.  It does work in UWP.

## Expected behavior

Popup displays in Wasm.

## How to reproduce it (as minimally and precisely as possible)

Attaching a sample solution.  Just run the wasm/uwp projects and click the button.
[App1.zip](https://github.com/unoplatform/uno/files/5643399/App1.zip)

## Environment

Nuget Package:

- [ ] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [x] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [x] Visual Studio 2019 (version: )
- [ ] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by davidjohnoliver** [comment]
Thanks for the report!

The underlying issue here is that Uno's `Popup` is in the wrong namespace: https://github.com/unoplatform/uno/issues/134 It'll be fixed as part of the next major version, as it involves a binary breaking change.

As a workaround, you can use the following:

```csharp
#if NETFX_CORE
using _Popup = Windows.UI.Xaml.Controls.Primitives.Popup;
#else
using _Popup = Windows.UI.Xaml.Controls.Popup;
#endif

...

    public partial class TestPopup : Control
    {
        _Popup _popup;

...

            _popup = GetTemplateChild("PART_Popup") as _Popup;
```

with that, the sample supplied should work.

**Comment by robloo** [comment]
Breaking changes rollup #2240

**Comment by MartinZikmund** [comment]
Fixed by #4695
